app/services/default_proxy_proxy_selector.rb

```diff
@@ -0,0 +1,18 @@
+class DefaultProxyProxySelector < ::RemoteExecutionProxySelector
+  def initialize
+    # TODO: Remove this once we have a reliable way of determining the internal proxy without katello
```

Comment:
Internal proxy is IMHO a concept we should not introduce in Foreman and I want to kill in Katello.